types/src/validator_verifier.rs

```diff
@@ -94,6 +94,7 @@ impl ValidatorVerifier {
         } else {
             total_voting_power * 2 / 3 + 1
         };
+
```

Comment:
please remove this change.

Reply:
Done (the formatting change was accidentally introduced).